Units/aiunits/ecommanderbattleai.lua

```diff
@@ -1,329 +1,329 @@
-		turret                   = true,
-		weaponVelocity           = 1000,
-		waterweapon				 = true,
-		customparams             = {
-			damagetype		     = "ecommanderbattle", 
-			nocosttofire		 = true,
-		}, 
-		damage                   = {
-			default              = weapon2Damage,
-		},
-	},
-}
-unitDef.weaponDefs               = weaponDefs
---------------------------------------------------------------------------------
-
-return lowerkeys({ [unitName]    = unitDef })
-
---------------------------------------------------------------------------------
+-- UNITDEF -- ECOMMANDERBATTLEAI --
+--------------------------------------------------------------------------------
+
+local unitName                   = "ecommanderbattleai"
+
+--------------------------------------------------------------------------------
+
+local armortype					 = [[armored]]
+local supplyGiven				 = [[200]]
+local techprovided				 = [[tech0, tech1, tech2, tech3, -overseer]]
+local techrequired				 = [[0 overseer]]
+
+local weapon1Damage              = 200
+local weapon1AOE				 = 250
+local energycosttofire			 = 0 --weapon1Damage / 10 * ((weapon1AOE / 1000) + 1)
+
+local weapon2Damage              = 25
+local weapon2AOE				 = 10
+local energycosttofire2			 = weapon2Damage / 10 * ((weapon2AOE / 1000) + 1)
+
+
+local unitDef                    = {
+
+	--mobileunit 
+	transportbyenemy             = false; 
+
+	--**
+
+	acceleration                 = 0.2,
+	brakeRate                    = 1,
+	buildCostEnergy              = 0,
+	buildCostMetal               = 500,
+	buildDistance                = 1000,
+	builder                      = true,
+	buildTime                    = 25,
+	capturable		             = false,
+	CanAttack			         = true,
+	CanAssist			         = true,
+	canBeAssisted                = true,
+	CanCapture                   = true,
+	CanRepair			         = true,
+	canRestore					 = false,
+	
+	canGuard                     = true,
+	canMove                      = true,
+	canPatrol                    = true,
+	canreclaim		             = false,
+	canstop                      = true,
+	category                     = "NOTAIR SUPPORT ARMORED",
+	description                  = [[Builds Units • Provides support in battles]],
+	energyMake                   = 0,
+	energyStorage                = 0,
+	energyUse                    = 0,
+	explodeAs                    = "commnuke",
+	footprintX                   = 4,
+	footprintZ                   = 4,
+	hideDamage		             = true,
+	iconType                     = "commander",
+	idleAutoHeal                 = .5,
+	idleTime                     = 2200,
+	levelground                  = true,
+	maxDamage                    = 10000,
+	maxSlope                     = 180,
+	maxVelocity                  = 3,
+	maxReverseVelocity           = 1,
+	maxWaterDepth                = 5000,
+	metalmake                    = 0,
+	metalStorage                 = 0,
+	movementClass                = "NORMALTANK2",

```

Comment:
This should be "TANK4"

Footprint size matters and putting a unit with a hitsphere of 4 into a pathing space of 2 will not work out nicely.